.github/workflows/windows_arm64.yml

```diff
@@ -5,6 +5,7 @@ on:
 
 env:
   python_version: 3.12
+  open_blas_link: "https://woastorage.blob.core.windows.net/packages/openblas/openblas_v0.3.27.zip"
```

Comment:
Unfortunately I think this might be a security hazard, linking against a binary blob whose provenance we can't keep track of.

Reply:
Ok. I expected similar response, but I wanted to check what do you think about this, as I couldn't find appropriate Fortran cross compiler binary. This is the reason I extracted the link to be obvious.
Gcc toolchain for cross-compilation should be available in a few months, but that would be the time when native builders would become available. In fact native Win and Linux arm64 runners should already be available for some customers, but I am not sure what is the NumPy community billing plan, and I have free GitHub account so I cannot see those those.
I compiled the version of the flang-new and added a few additional binaries to make it work, but this build is taking several hours so it is not a solution for the NumPy. I also built  win arm64 openblas natively to test the performance but that is also not a solution here. 

 I will rethink if there is some different solution.

Reply:
Maybe it would be more appropriate to discuss this at https://github.com/MacPython/openblas-libs. That is where the scipy-openblas wheels are built. We already cross-compile under a qemu docker for aarch64 (which takes over 2 hours), we could do something similar for arm64-windows if there is a way to do that.